client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -117,12 +110,12 @@ const NotificationSubscriptions = createReactClass( {
 								{ this.props.translate( 'Default email delivery' ) }
 							</FormLabel>
 							<FormSelect
-								disabled={ this.getDisabledState() }
+								disabled={ this.props.getDisabledState() }
 								id="subscription_delivery_email_default"
 								name="subscription_delivery_email_default"
-								onChange={ this.updateSetting }
+								onChange={ this.props.updateSetting }
 								onFocus={ this.handleFocusEvent( 'Default Email Delivery' ) }
-								value={ this.getSetting( 'subscription_delivery_email_default' ) }
+								value={ this.props.getSetting( 'subscription_delivery_email_default' ) }
```

Comment:
This looks like a mistake, `getSetting` is actually a method and not a prop:

```suggestion
								value={ this.getSetting( 'subscription_delivery_email_default' ) }
```

This is observed at multiple locations in this file.

Reply:
`getSetting` is a method from the `withFormBase` hoc which passes it down as a prop to `NotificationSubscriptions` so that should be correct.